account_payment_order/wizard/account_payment_line_create_view.xml

```diff
@@ -19,7 +19,10 @@
                 <field name="due_date" attrs="{'required': [('date_type', '=', 'due')], 'invisible': [('date_type', '!=', 'due')]}"/>
                 <field name="journal_ids"
                    widget="many2many_tags"
-                    placeholder="Keep empty for using all journals"/>
+                   placeholder="Keep empty to use all journals"/>
```

Comment:
Don't modify this placeholder due to translations and it was correct previously also